auth-web/src/views/BusinessProfile.vue

```diff
@@ -2,7 +2,7 @@
   <div class="view-container">
     <article>
       <h1>Update Business Profile</h1>
-      <p class="intro-text">{{ $t('businessContactCopy', { businessType })}}</p>
+      <p class="intro-text">It looks like we are missing some contact information for your {{businessType}}. You will need to supply us with a few additional details before you can get started...</p>
```

Comment:
Are we going to forego localization for now? I'm fine with it if that's the case, just making sure.

Reply:
Nope..We will use it..  It has to do something with the Vuetify added in the common-common-components.Weird behavior. I am working on it.